tsb/TSB_Loader/src/com/fairwaytech/utilities/XmlUtilities.java

```diff
@@ -98,7 +99,11 @@ public static TestSpecBankRegistration Generate(File file, String tenantId) {
         return result;
     }
 
-    private static void replaceTestNameWithUniqueId(final File file) {
+    public static byte[] documentToByte(Document document)
+    {
```

Comment:
I know this doesn't matter but can we put the curly on line 102 with the method?